apps/meteor/server/services/team/service.ts

```diff
@@ -473,7 +474,7 @@ export class TeamService extends ServiceClassInternal implements ITeamService {
 		await Rooms.setTeamDefaultById(rid, isDefault);
 
 		if (room.teamDefault) {
-			const teamMembers = await this.members(uid, room.teamId, true, undefined, undefined);
+			const teamMembers = await this.members(uid, room.teamId, true, { offset: 0, count: MAX_NUMBER_OF_AUTO_JOIN_MEMBERS });
```

Comment:
Here it also returns the total count of members, we can return this total members count from this function and use above.